codegen/templates/vec.rs.tera

```diff
@@ -1074,6 +1074,48 @@ impl {{ self_t }} {
     pub fn distance_squared(self, rhs: Self) -> {{ scalar_t }} {
         (self - rhs).length_squared()
     }
+
+    /// Returns the element-wise quotient of [Euclidean division] of `self` by `rhs`.
+    ///
+    {% if is_float%}
+    {% if not is_scalar %}
```

Comment:
If you look at the generated output from this there are some unwanted newlines, you can remove those by adding a `-` to `{%` or `%}`, e.g. `{%-`.